packages/renderer/src/lib/agent-workspaces/agent-utis.ts

```
@@ -0,0 +1,29 @@
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { isDark } from '/@/stores/appearance';
+import type { AgentInfo } from '/@api/agent-info';
+
+export function getAgentIcon(agentInfo: AgentInfo | undefined): string {
+  const icon = agentInfo?.icon?.icon;
+  if (typeof icon === 'string') {
+    return icon;
+  } else if (typeof icon === 'object') {
+    return isDark ? icon.dark : icon.light;
+  }
```

Biome (2.5.0)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)